runelite-client/src/main/java/net/runelite/client/plugins/timetracking/TimeTrackingPlugin.java

```diff
@@ -174,6 +171,16 @@ else if (clockManager.getStopwatches().isEmpty() && e.getKey().equals(STOPWATCHE
 		}
 	}
 
+	@Subscribe
+	public void onCommandExecuted(CommandExecuted commandExecuted)
+	{
+		if (commandExecuted.getCommand().equals("resetfarmtick"))
```

Comment:
Should this be behind developer mode?

Reply:
I think it is important for users to be able to reset their timing if anything goes wrong.